uk_bin_collection/uk_bin_collection/councils/MedwayCouncil.py

```
@@ -20,7 +20,7 @@ def parse_data(self, page: str, **kwargs) -> dict:
 
         api_url = f"https://api.medway.gov.uk/api/waste/getwasteday/{user_uprn}"
 
-        response = requests.get(api_url)
+        response = requests.get(api_url, verify=False)
```

Ruff (0.14.7)

Probable use of `requests` call without timeout

(S113)